Readme.md

```diff
@@ -42,6 +42,78 @@ $ npm install koa
  - [Introduction Screencast](http://knowthen.com/episode-3-koajs-quickstart-guide/) - An introduction to installing and getting started with Koa
 
 ## Example
+```js
+const Koa = require('koa');
+const app = new Koa();
+
+// logger
+
+app.use(function (ctx, next){
+  const start = new Date;
+  return next().then(function() {
+    const ms = new Date - start;
+    console.log(`${ctx.method} ${ctx.url} - ${ms}`);
+  });
+});
+
+// response
+
+app.use(function (ctx){
+  ctx.body = 'Hello World';
+});
+
+app.listen(3000);
+```
+
+## Example with asyncFunction(babel-node required)
+
+```js
+const Koa = require('koa');
+const app = new Koa();
+
+// logger
+
+app.use(async (ctx, next) => {
+  const start = new Date;
+  await next();
+  const ms = new Date - start;
+  console.log(`${ctx.method} ${ctx.url} - ${ms}`);
+});
+
+// response
+
+app.use(function (ctx){
+  ctx.body = 'Hello World';
+});
+
+app.listen(3000);
+```
+
+## Example with generator
```

Comment:
let's note here that this is a stop-gap solution - it will eventually be removed when `async` functions are native.